docs/how-to/integrate_sdcore_with_external_gnb.md

```diff
@@ -0,0 +1,60 @@
+# Integrate SD-Core with an Externally Managed Radio
+
+For simplicity in managing deployments, the gNB Name and TAC can be supplied via a charm integration. This is the purpose of the sdcore-gnb-integrator charm.
+
+## Pre-requisites
+
+- A Kubernetes cluster capable of handling the load from both SD-Core and one additional container per represented gNB
```

Comment:
Re-reading this, it doesn't make sense to force the gNB integrator to the same k8s cluster as the core, so maybe this is not needed?